ci-operator/step-registry/telco5g/origin/tests/telco5g-origin-tests-ref.yaml

```diff
@@ -10,15 +10,23 @@ ref:
     requests:
       cpu: 100m
       memory: 200Mi
+  credentials:
+  - namespace: test-credentials
+    name: cikey
+    mount_path: /var/run/ci-key
   env:
-  - name: E2E_TESTS_CONFIG
+  - name: TEST_SUITE
+    default: openshift/conformance/parallel
+    documentation: The test suite to run.  Use 'openshift-test run --help' to list available suites.
+  - name: TEST_SKIPS
     default: ""
-    documentation: Additional configuration.
+    documentation: |
```

Comment:
So, skips or white list?

Reply:
Others use skips and we have ~5 failing and 5-10 flaky tests so i think its feasible to use a blacklist.